js_modules/dagster-ui/packages/ui-core/src/app/getVisibleFeatureFlagRows.tsx

```diff
@@ -24,6 +24,10 @@ export const getVisibleFeatureFlagRows = () => [
     key: 'Use new asset automation page',
     flagType: FeatureFlag.flagUseNewAutomationPage,
   },
+  {
```

Comment:
Can you check cloud to make sure this doesn't result in 2 of these entries?

Reply:
Removed it here just to reduce confusion: https://github.com/dagster-io/internal/pull/8976